metrics/metrics.go

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// IP Geolocation by DB-IP (https://db-ip.com)
```

Comment:
Please remove this. This repository has no dependency on DB-IP

Reply:
I put this in because legal requested I put attribution in "the file where we do the work"

Reply:
We can check with her again, but this is definitely not the place. Anyone can be using this code with whatever db they want. If anything, the comment should be in the Dockerfile. or the update script.

Reply:
I'll clarify with her in the email thread and come back around

Reply:
From offline conversation, we'll put this comment in the db fetch script instead.

Reply:
Updated the PR in outline-server